remote-config.jsonc

```
@@ -93,6 +93,36 @@
                 },
 				{
                     "message": "MacStadium supports DDEV with an M1 Mac Mini that runs macOS tests"
+                },
+                {
+                    "message": "You can turn off the cell borders in `ddev describe` and `ddev list` with `ddev config global --simple-formatting`"
+                },
+                {
+                    "message": "You can use a more lively `ddev list` and `ddev describe` output with `ddev config global --table-style=bright`. Try it out!"
+                },
+                {
+                    "message": "DDEV's fiscal entity is now the DDEV Foundation. http://ddev.com/foundation. We have a real focus on the long-term sustainability of the project, not just depending on Randy."
+                },
+                {
+                    "message": "Tag1 Consulting has become a major sponsor of DDEV with $1K/month! Stop by and thank them, https://tag1.com."
```

I’d make the URL part of the sentence, instead of having the comma add an unnecessary pause without any supporting language:

```suggestion
                    "message": "Tag1 Consulting has become a major sponsor of DDEV with $1K/month! Stop by and thank them at https://tag1.com."
```